ui/pages/confirmation/templates/add-ethereum-chain.js

```diff
@@ -32,6 +32,7 @@ const UNRECOGNIZED_CHAIN = {
 
 const MISMATCHED_CHAIN_RECOMMENDATION = {
   id: 'MISMATCHED_CHAIN_RECOMMENDATION',
+  severity: SEVERITIES.WARNING,
```

Comment:
I noticed this was missing while running snapshot tests, adding this will remove some required prop warning.